dashboard/lib/tasks/seed.rake

```diff
@@ -148,15 +147,16 @@ namespace :seed do
     update_scripts(script_files: script_files)
   end
 
-  task scripts: SCRIPTS_DEPENDENCIES do
+  task scripts: :environment do
+    time_subtasks(SCRIPTS_DEPENDENCIES)
```

Comment:
It looks like there is more going on here than what's listed in the PR description. why are we timing this? why only for this rake target and not the next two?

Reply:
Updated PR description.

And good point on the inconsistency - if you're on board with this change overall I'll apply it to the next two also (or I'll get rid of it for all three).